### DISCUSSION TITLE
What is your comments on applying N4BiasFieldCorrection on 4D functional images?

### DISCUSSION THREAD

**Discussion by WangRoss** [ANTsX/ANTs#1286] [Q&A]
I want to use N4BiasFieldCorrection on 4D functional images with the
aim of reducing inhomogeneity in signal intensity resulting from the
bias field. I am concerned about 3 questions as follows:

1. The placement of the detection coils contribute to the bias field,
but for the same subject, does the subject's head placement or
movement influce the bias field?

2. I first applied N4BiasFieldCorrection on 4D raw data of functional
images, but I am not sure whether the operation would change the shape
of voxel time series? I found some researchers performed the N4
correction on the mean functional image (after motion correction and
slice timing). I am confused that is the N4 corection meaningful when
the preprocessing might change something of the data, e.g.,
distribution?
On my experience, the effect on the raw data is much better than doing
on the preprocessed functional data. But I have no idea about the
underlying principles.

Would you please give a standard protocol for correcting 4D functional data?

3.  Or, it is unneccessary to do the process on the functional time series?

What is your comments on applying N4BiasFieldCorrection on 4D functional images?

**Comment by ntustison** [comment]
> The placement of the detection coils contribute to the bias field, but for the same subject, does the subject's head placement or movement influce the bias field?

Yes.

> I first applied N4BiasFieldCorrection on 4D raw data of functional images, but I am not sure whether the operation would change the shape of voxel time series? I found some researchers performed the N4
correction on the mean functional image (after motion correction and slice timing). I am confused that is the N4 corection meaningful when the preprocessing might change something of the data, e.g.,
distribution? On my experience, the effect on the raw data is much better than doing on the preprocessed functional data. But I have no idea about the underlying principles. 

> Would you please give a standard protocol for correcting 4D functional data?

> Or, it is unneccessary to do the process on the functional time series?  What is your comments on applying N4BiasFieldCorrection on 4D functional images?

Bias correction can be used.  For ANTs-based pipelines, both N3 and N4 have been used.  However, it's oftentimes a personal preference.  If you find that bias correction does not improve your results, than you shouldn't use it.

**Reply by WangRoss** [reply]
Thank you very much!
Acturally I am converned about whether N4 could be used on raw 4D-BOLD images or after some basic preprocessing (e.g., motion correction, slicetiming), which choice is better. I found it would change much if the correction is applied on raw data while it may not show much difference when the operation was used on the slicetiming corrected BOLD images.
Would you like to share your some understanding?
Great thanks!
